src/components/Event/docs/index.story.tsx

```diff
@@ -29,9 +29,19 @@ export default {
   componentPath: '../Event.tsx',
   componentProps: () => ({
     'data-hook': 'storybook-Event',
+    time: '21:30',
+    title: 'Yoga Class',
+    showTime: true,
+    fullday: false,
+    selected: false,
+    disabled: false,
+    onClick: undefined,
```

Comment:
never used this part before... I'll use it in future TPA components

Reply:
🏆